sdk/halo-framework/src/main/java/com/mobgen/halo/android/framework/api/HaloFramework.java

```diff
@@ -199,6 +204,27 @@ public boolean isInDebugMode() {
     }
 
     /**
+     * Sets the print policy flag on the framework.
+     *
+     * @param printPolicy The print to file policy
+     */
+    @Api(2.2)
+    public void setPrintToFilePolicy(int printPolicy){
```

Comment:
This name is not refering to the log anyway. Please change the name so it will be descriptive enough in that sense